src/server/api/routes/users/v2/index.js

```diff
@@ -0,0 +1,13 @@
+const { Router } = require('express');
+
+const auth = require('./auth');
+const follow = require('./follow');
+const nfts = require('./nfts');
+
+const router = Router();
+
+router.use(auth);
+router.use(follow);
+router.use(nfts);
```

Comment:
If we are doing all these nested Router, we should put the common path `v2/users` here instead of in each route?
Also not sure if nested router affects performance

Reply:
Agree. In the beginning. I also wondered if we should put those APIs with `/:wallet` part in another subfolder. Probably `_wallet`? (similar to the file structure of Nuxt)
- Pros:
  - File structure exactly reflects routes
  - Can use common path in `router.use()`, no need to repeat in each route. 
- Cons:
  - Files sink down a little bit deeper
  - `/:wallet` doesn't show on each route, may need some time to get to the point.

BTW, didn't find any issue or discussion about Express nested routes after some simple search.

Reply:
OK, does it means that we got this:
```suggestion
router.use(auth);
router.use('/:wallet', follow);
router.use('/:wallet/nfts', nfts);
```
or this:
```suggestion
router.use(auth);
router.use('/:wallet', wallet);
```
and create a `/_wallet/index.js` file as an entry for all routes with `/:wallet`?

Reply:
So we still keep the current state for now, right?